core/src/View/Kokkos_ViewTraits.hpp

```diff
@@ -32,6 +32,37 @@ static_assert(false,
 //----------------------------------------------------------------------------
 
 namespace Kokkos {
+
+struct ALL_t {
+  KOKKOS_INLINE_FUNCTION
```

Comment:
How about we change KOKKOS_INLINE_FUNCTION -> KOKKOS_FUNCTION while we are at it?